src/lift.js

```diff
@@ -18,11 +18,11 @@ var liftN = require('./liftN');
  *
  *      var madd3 = R.lift((a, b, c) => a + b + c);
  *
- *      madd3([1,2,3], [1,2,3], [1]); //=> [3, 4, 5, 4, 5, 6, 5, 6, 7]
+ *      // computes Cartesian product for Arrays
+ *      madd3([100, 200], [30, 40, 50], [6, 7]); //=> [136, 137, 146, 147, 156, 157, 236, 237, 246, 247, 256, 257]
```

Comment:
maybe something like :

"In Array context, the `lift`-ed function applies over the Cartesian product of the argument Arrays."

?